modules/protobuf/src/main/java/com/spotify/ffwd/protobuf/ProtobufDecoder.java

```diff
@@ -145,18 +141,4 @@ private Object decodeMetric0(final Protocol0.Metric metric) {
 
         return attributes;
     }
-
-    private Object decodeEvent0(final Protocol0.Event event) {
```

Comment:
ah we have to remove event from https://github.com/spotify/ffwd-client-java too

Reply:
Created https://github.com/spotify/ffwd-client-java/issues/3 to do that